components/x-interaction/__tests__/registerComponent.test.js

```diff
@@ -0,0 +1,54 @@
+const {
```

Comment:
**nitpick**: this file would be a bit more readable with some more whitespace e.g. between `it` blocks and after the `requires`